shell/components/form/Labels.vue

```diff
@@ -52,7 +52,18 @@ export default {
     showLabelTitle: {
       type:    Boolean,
       default: true,
-    }
+    },
+
+    showLabelDescription: {
+      type:    Boolean,
+      default: true,
+    },
+
+    // A custom label description. Defaults to `labels.labels.description` if empty.
+    labelDescription: {
+      type:    String,
+      default: '',
```

Comment:
Would this make sense?
```suggestion
      default: 'labels.labels.description',
```